src/main/java/com/gooddata/md/report/Total.java

```diff
@@ -41,4 +44,16 @@ public static Total of(String total) {
                     e);
         }
     }
+
+    public static List<Total> orderedValues() {
+        List<Total> orderedTotals = new ArrayList<>();
```

Comment:
user rather `Arrays.asList`

Reply:
and also add javadoc explaining this refers the order the totals are returned by executor

Reply:
changed & added javadoc